CHANGELOG.md

```diff
@@ -4,8 +4,8 @@
 ### Feature
 - User can soft delete patient
 - Add diagnosis and blood sugar readings to the teleconsultation message
+- Support multiple teleconsultation phone numbers
```

Comment:
Can we reword this to better describe a user facing feature?

Ex: "User can now select from multiple doctors when requesting a teleconsultation".